Mathlib/Condensed/Explicit.lean

```diff
@@ -29,16 +29,15 @@ continuous surjection), the presheaf `F` exhibits `F(B)` as the equalizer of th
 
 universe v u
 
-open CategoryTheory Limits Opposite Functor Presieve regularCoverage
+open CategoryTheory Limits Opposite Functor Presieve regularTopology
```

Comment:
See my comments in the first file about this renaming

Reply:
Resolving this to keep the discussion about the renaming on the first comment